ConceptMap/ConceptMap-Genomics-specimen-final-sample.json

```diff
@@ -0,0 +1,136 @@
+  "date": "2024-08-16T18:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording of concepts for the categories of final sample types.This is intended for the element Specimen.type",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "targetUri": "http://snomed.info/sct",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "DNA",
+          "display": "DNA",
+          "target": [
+            {
+              "code": "258566005",
+              "display": "Deoxyribonucleic acid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "DNA/RNA",
+          "display": "DNA/RNA",
+          "target": [
+            {
+              "code": "258566005",
+              "display": "Deoxyribonucleic acid specimen (specimen)",
+              "equivalence": "narrower"
+            },
+            {
+              "code": "441673008",
+              "display": "Ribonucleic acid specimen (specimen)",
+              "equivalence": "narrower"
+            }
+          ]
+        },
+        {
+          "code": "RNA",
+          "display": "RNA",
+          "target": [
+            {
+              "code": "441673008",
+              "display": "Ribonucleic acid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "cDNA",
+          "display": "cDNA",
+          "target": [
+            {
+              "code": "70016006",
+              "display": "Complementary deoxyribonucleic acid (substance)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ctDNA",
+          "display": "ctDNA",
+          "target": [
+            {
+              "equivalence": "unmatched"
```

Comment:
Add comment to state how this should be recorded, e.g. recorded under Specimen.type as free text "Circulating Tumour DNA"